src/frontends/onnx/frontend/src/op/conv.cpp

```diff
@@ -60,17 +61,21 @@ OutputVector conv(const Node& node,
 
     // no bias param
     if (ngraph::op::is_null(bias)) {
+        common::set_friendly_names(conv_node, nullptr, node);
         return {conv_node};
     } else {
         const auto& bias_ps = bias.get_partial_shape();
 
         NGRAPH_CHECK(bias_ps.rank().is_static() && bias_ps.rank().get_length() == 1,
                      "The bias input needs to be 1D vector");
 
-        return {add_bias(conv_node, bias)};
+        const auto bias_node = add_bias(conv_node, bias);
+        common::set_friendly_names(conv_node, bias_node, node);
```

Comment:
this can be simplified a bit. You can just call conv_node->set_friendly_name here and the name for bias_node would be handled in Graph::set_friendly_names. Therefore we wouldn't have to set or get SKIP_SETTING_FRIENDLY_NAME property.